### ISSUE TITLE
Make cross-platform

### ISSUE THREAD

**Issue by antony** [appliedblockchain/cobalt#17]
There are probably some other concerns, but a couple of notable ones:

1. Include `apt` or similar instructions in the readme
2. Code like this:

```js
const { web3, accounts } = require('../lib/web3')({
  root: `${__dirname}/../contracts`
})
```

should be changed to:

```js
const { join } = require('path')

const { web3, accounts } = require('../web3')({
  root: join(__dirname, '..', 'contracts')
})
```

**Comment by mirek** [comment]
Fixed in #18

**Comment by mirek** [comment]
Merged linter branch; the other one - it's in the root directory on purpose so you can cherry pick what you're interested in and your require looks good without diving into internal structure.
